### PR TITLE
bus-util: add several macros for defining functions of getting dbus properties

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/shared/bus-util.h [hunk 1]
```diff
@@ -122,6 +122,59 @@ assert_cc(sizeof(mode_t) == sizeof(uint32_t));
...
+                data_type *data = userdata;                             \
+                                                                        \
+                assert(bus);                                            \
+                assert(reply);                                          \
+                assert(data);                                           \
+                                                                        \
+                return sd_bus_message_append(reply, bus_type,           \
+                                             get2(get1(data)));         \
+        }
+
```
Comment (keszybz) [inline src/shared/bus-util.h:158]: What about defining
```C
#define ident(x) (x)
#define BUS_DEFINE_PROPERTY_GET(function, bus_type, data_type, get)     \
  BUS_DEFINE_PROPERTY_GET2(function, bus_type, data_type, get1, ident)
#define BUS_DEFINE_PROPERTY_GET_GLOBAL(function, bus_type, val)         \
  BUS_DEFINE_PROPERTY_GET(function, bus_type, data_type, ident)
```
?

Comment (yuwata) [reply src/shared/bus-util.h:158]: Hmm. I have no idea how to define `BUS_DEFINE_PROPERTY_GET_GLOBAL()` by `BUS_DEFINE_PROPERTY_GET()`...

**Comment by yuwata** [comment]
@keszybz Updated. The macro `BUS_DEFINE_PROPERTY_GET()` is redefined by `_GET2()`.

File: src/shared/bus-util.h (hunk 1 above)
Comment (keszybz) [reply src/shared/bus-util.h:158]: Oh, OK. Yeah, I missed the fact that they take different args. Looks good now.

**Pull request merged**